models/usersActivity.model.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+
+const UsersActivitySchema = new mongoose.Schema({
+    _id:{type:mongoose.Schema.Types.ObjectId,ref:"Users"},
+    wishlist:[{type:Schema.Types.ObjectId,ref:'Books'}],
+    cart:[{count:{type:Number},productId:{type:Schema.Types.ObjectId,ref:'Books'}}]
```

Comment:
quantity would be better instead of count right. It would be a domain-specific name?